tests/examples/test_example_bots_training_data.py

```diff
@@ -1,50 +1,59 @@
+import warnings
 from pathlib import Path
-from typing import Text
+from typing import Optional, Text
 
 import pytest
 
 from rasa.cli import scaffold
 from rasa.shared.importers.importer import TrainingDataImporter
+from tests.conftest import filter_expected_warnings
 
 
 @pytest.mark.parametrize(
-    "config_file, domain_file, data_folder, raise_slot_warning",
+    "config_file, domain_file, data_folder, raise_slot_warning, msg",
     [
         (
             "examples/concertbot/config.yml",
             "examples/concertbot/domain.yml",
             "examples/concertbot/data",
             True,
+            None,
         ),
         (
             "examples/formbot/config.yml",
             "examples/formbot/domain.yml",
             "examples/formbot/data",
             True,
+            None,
         ),
         (
             "examples/knowledgebasebot/config.yml",
             "examples/knowledgebasebot/domain.yml",
             "examples/knowledgebasebot/data",
             True,
+            "You are using an experiential feature: "
```

Comment:
shouldn't it be "experimental"?

Reply:
Indeed, I'll correct the incorrect word.

Reply:
Addressed: https://github.com/RasaHQ/rasa/pull/11773/commits/bfd5135e0d7127da726a92b50602c29ae1991636